quickstart/IntroNotebooks/4. Using PyTorch through ONNX.ipynb

```diff
@@ -218,174 +575,159 @@
+      "[04/06/2021-04:19:19] [I] Format: ONNX\n",
+      "[04/06/2021-04:19:19] [I] Model: resnet50_pytorch.onnx\n",
+      "[04/06/2021-04:19:19] [I] Output:\n",
+      "[04/06/2021-04:19:19] [I] === Build Options ===\n",
+      "[04/06/2021-04:19:19] [I] Max batch: explicit\n",
+      "[04/06/2021-04:19:19] [I] Workspace: 16 MB\n",
+      "[04/06/2021-04:19:19] [I] minTiming: 1\n",
+      "[04/06/2021-04:19:19] [I] avgTiming: 8\n",
+      "[04/06/2021-04:19:19] [I] Precision: FP32+FP16\n",
+      "[04/06/2021-04:19:19] [I] Calibration: \n",
+      "[04/06/2021-04:19:19] [I] Safe mode: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Save engine: resnet_engine_pytorch.trt\n",
+      "[04/06/2021-04:19:19] [I] Load engine: \n",
+      "[04/06/2021-04:19:19] [I] Builder Cache: Enabled\n",
+      "[04/06/2021-04:19:19] [I] NVTX verbosity: 0\n",
+      "[04/06/2021-04:19:19] [I] Input: fp16:chw\n",
+      "[04/06/2021-04:19:19] [I] Output: fp16:chw\n",
+      "[04/06/2021-04:19:19] [I] Input build shapes: model\n",
+      "[04/06/2021-04:19:19] [I] Input calibration shapes: model\n",
+      "[04/06/2021-04:19:19] [I] === System Options ===\n",
+      "[04/06/2021-04:19:19] [I] Device: 0\n",
+      "[04/06/2021-04:19:19] [I] DLACore: \n",
+      "[04/06/2021-04:19:19] [I] Plugins:\n",
+      "[04/06/2021-04:19:19] [I] === Inference Options ===\n",
+      "[04/06/2021-04:19:19] [I] Batch: Explicit\n",
+      "[04/06/2021-04:19:19] [I] Input inference shapes: model\n",
+      "[04/06/2021-04:19:19] [I] Iterations: 10\n",
+      "[04/06/2021-04:19:19] [I] Duration: 3s (+ 200ms warm up)\n",
+      "[04/06/2021-04:19:19] [I] Sleep time: 0ms\n",
+      "[04/06/2021-04:19:19] [I] Streams: 1\n",
+      "[04/06/2021-04:19:19] [I] ExposeDMA: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Spin-wait: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Multithreading: Disabled\n",
+      "[04/06/2021-04:19:19] [I] CUDA Graph: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Skip inference: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Inputs:\n",
+      "[04/06/2021-04:19:19] [I] === Reporting Options ===\n",
+      "[04/06/2021-04:19:19] [I] Verbose: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Averages: 10 inferences\n",
+      "[04/06/2021-04:19:19] [I] Percentile: 99\n",
+      "[04/06/2021-04:19:19] [I] Dump output: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Profile: Disabled\n",
+      "[04/06/2021-04:19:19] [I] Export timing to JSON file: \n",
+      "[04/06/2021-04:19:19] [I] Export output to JSON file: \n",
+      "[04/06/2021-04:19:19] [I] Export profile to JSON file: \n",
+      "[04/06/2021-04:19:19] [I] \n",
       "----------------------------------------------------------------\n",
       "Input filename:   resnet50_pytorch.onnx\n",
       "ONNX IR version:  0.0.6\n",
       "Opset version:    9\n",
       "Producer name:    pytorch\n",
-      "Producer version: 1.8\n",
+      "Producer version: 1.6\n",
```

Comment:
Torch version also changed?

Reply:
I updated to 1.9 to verify this works with latest 21.05 NGC containers. Is that okay?

Reply:
Yes, thanks.